src/request.rs

```diff
@@ -16,6 +17,9 @@ use std::sync::Arc;
 use std::time::Duration;
 
 const LOCK_RETRY_DELAY_MS: u64 = 10;
+const REGION_RETRY_BASE_DELAY_MS: u64 = 2;
+const REGION_RETRY_MAX_DELAY_MS: u64 = 500;
+const REGION_RETRY_ATTEMPTS: u32 = 10;
```

Comment:
I think we can just declare a const `NoJitterBackoff` for it.

Reply:
Hi @sticnarf , I replace these constants to one `const NoJitterBackoff`, and fix initializing this struct directly in other places (the compiler prompted the `new` function must be `const`, it looks a little strange, so I remove the `new` function).

And I update other places based on your code review. Please help review. Thanks.

Reply:
I think the right approach is just declare the `new` function as a `const fn`. `const fn`s are used to return a const value.

Reply:
Hi @sticnarf , I rollbacks the `new` function of struct NoJitterBackoff, and adds `const`. Please help review. Thanks.